### PR TITLE
Redesign - Run detail 

### PR DESCRIPTION
**Pull request by jozefbakus**
implements: #88

### CODE REVIEW

File: ui/src/app/components/runs/run-detail/run-detail.component.html [hunk 1]
```diff
@@ -13,9 +13,7 @@
   ~ limitations under the License.
   -->
 
-<div class="content-area">
-  <clr-datagrid (clrDgRefresh)="refresh($event)" [clrDgLoading]="loading">
-    <clr-dg-column>ID</clr-dg-column>
+  <clr-datagrid [clrDgLoading]="loading">
     <clr-dg-column>Name</clr-dg-column>
```
Comment (kevinwallimann) [inline ui/src/app/components/runs/run-detail/run-detail.component.html]: I would rename the column header to `Job Name`

File: ui/src/app/components/runs/runs.component.html [hunk 2]
```diff
@@ -60,7 +60,13 @@
   </clr-dg-row>
 
   <clr-dg-detail *clrIfDetail="let detail">
-    <clr-dg-detail-header>{{detail.workflowName}} ({{detail.projectName}}) - {{detail.started | date:'medium'}} - {{detail.finished | date:'medium'}}</clr-dg-detail-header>
+    <clr-dg-detail-header>
```
Comment (kevinwallimann) [inline ui/src/app/components/runs/runs.component.html]: Hmm.. It looks a bit "in your face", tbh. But I don't have a better idea right now

File: ui/src/app/components/runs/run-detail/run-detail.component.html (hunk 1 above)
Comment (jozefbakus) [reply ui/src/app/components/runs/run-detail/run-detail.component.html]: Done

File: ui/src/app/components/runs/runs.component.html (hunk 2 above)
Comment (jozefbakus) [reply ui/src/app/components/runs/runs.component.html]: Yes, bit. We can take a look on this later.

Comment (kevinwallimann) [reply ui/src/app/components/runs/runs.component.html]: Ok

**Review by kevinwallimann — APPROVED** [review]

**Pull request merged**